src/otg/dataset/study_index.py

```diff
@@ -459,10 +459,10 @@ def from_source(
                     f.col("num_cases").alias("nCases"),
                     f.col("num_controls").alias("nControls"),
                     # Set constant value columns.
-                    f.lit("FINNGEN_R8").alias("projectId"),
+                    f.lit("FINNGEN_R9").alias("projectId"),
```

Comment:
Could this pick up the value of `finngen_release_prefix`?

Reply:
Thank you, I think it's a very good suggestion, now implemented: https://github.com/opentargets/genetics_etl_python/pull/88/commits/bb3ec04fa49763faf32d6daafb539b86a1ae527e